plugins/auth-backend/src/providers/oidc/provider.test.ts

```diff
@@ -89,27 +91,38 @@ describe('OidcAuthProvider', () => {
     const provider = new OidcAuthProvider(clientMetadata);
     const req = {
       method: 'GET',
-      url: '/?code=test2',
+      url: 'https://oidc.test/?code=test2',
       session: ({ 'oidc:oidc.test': 'test' } as any) as Session,
     } as express.Request;
     await provider.handler(req);
     expect(scope.isDone()).toBeTruthy();
   });
 
-  const options = {
-    globalConfig: {
-      appUrl: 'https://oidc.test',
-      baseUrl: 'https://oidc.test',
-    },
-    config: ({
-      keys: jest.fn(() => ['test']),
-      getConfig: jest.fn(() => ({ getString: () => '' })),
-    } as any) as Config,
-  } as AuthProviderFactoryOptions;
-
-  it('createOidcProvider', () => {
+  it('createOidcProvider', async () => {
+    const scope = nock('https://oidc.test')
```

Comment:
Also missed this one, we're using `msw` for this purpose throughout the codebase. Fine to leave this but if you wanna refactor that's 👌 too

Reply:
Can I do that in another PR (in a couple of days)?  Getting the nock `scope.isDone()` assertion to work seems like it will be non-trivial.

Reply:
Definitely!

If I understand the current usage correctly I think you'll be able to replace `isDone` with https://jestjs.io/docs/en/expect#expectassertionsnumber + expect in the handlers